limitations.md

```diff
@@ -0,0 +1,13 @@
+# Limitations in iceoryx
```

Comment:
When do we update this file? For every release?

Reply:
We should at least check of the limitation still apply then.

Reply:
How do you plan to update this document? It can get out of sync easily. Could you maybe add a release check list to the wiki, so that we don't forget this document?